src/redux/actions/index.test.js

```diff
@@ -0,0 +1,52 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import * as actions from './index';
+import * as types from '../constants/action-types';
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('actions', () => {
+  it('should change search option', () => {
+    const payload = 'title';
+    const expectedAction = {
+      type: types.CHANGE_SEARCH,
+      payload
+    };
+    expect(actions.changeSearch(payload)).toEqual(expectedAction);
+  });
+
+  it('should change sorting option', () => {
+    const payload = 'rating';
+    const expectedAction = {
+      type: types.CHANGE_SORTING,
+      payload
+    };
+    expect(actions.changeSorting(payload)).toEqual(expectedAction);
+  });
+
+  it('should trigger start search', () => {
+    const expectedAction = {
+      type: types.SEARCH_STARTED
+    };
+    expect(actions.searchStarted()).toEqual(expectedAction);
+  });
+
+  it('should trigger search failure', () => {
+    const expectedAction = {
+      type: types.SEARCH_FAILURE
+    };
+    expect(actions.searchFailure()).toEqual(expectedAction);
+  });
+
+  it('getSearchData should trigger searchStarted and searchSuccess', () => {
+    const expectedActions = [types.SEARCH_STARTED, types.SEARCH_SUCCESS];
+
+    const store = mockStore({ movies: [] });
+
+    return store.dispatch(actions.getSearchData()).then(() => {
```

Comment:
This action performs real side effect (call to API). It's better to mock such things, because this test can be started in isolated environment (with no access to API).
Also, looks like it should be called like:

```js
actions.getSearchData(<args>)(store.dispatch).then(...);
```